crates/phala-mq/src/types.rs

```diff
@@ -53,8 +60,18 @@ impl MessageOrigin {
     pub fn is_gatekeeper(&self) -> bool {
         matches!(self, Self::Gatekeeper)
     }
+
+    /// Returns the account id if the origin is from a user, or `Err(BadOrigin)` otherwise
+    pub fn account(self) -> Result<AccountId32, BadOrigin> {
```

Comment:
Suggest to rename it to `ensure_signed()` to match Substrate's convention

Reply:
> Suggest to rename it to `ensure_signed()` to match Substrate's convention

This is not on-chain code, So I think it's better to match Rust's convention.